boilerplate/app/i18n/LocaleContext.tsx

```diff
@@ -0,0 +1,34 @@
+import "../i18n"
+import React from "react"
+import I18n from "i18n-js"
+import { translate } from "./translate"
+import { useStorage } from "../utils/storage"
+import { UPDATE_LANGUAGE } from "../utils/constants"
+
+const LocaleContext = React.createContext<any>(null)
+
+export const LocaleContextProvider = (props) => {
+  const [locale, changeLocale] = useStorage(UPDATE_LANGUAGE, "en")
```

Comment:
Does this mean we lose the default usage of the system language, and hardcode english?

I just made a proposal tangentially related to this:
https://github.com/infinitered/ignite/pull/2415